documentation/FlowManager.md

```diff
@@ -185,3 +185,34 @@ OR if the last login/logout process for <mytoken> finished less than 5 seconds a
     }
 }
 ```
+
+#### Update settings on Online Flowmanager (PUT /:token/untrusted-settings)
+* **description**: Update preferences on the online flowmanager
+* **Supported modes**: Cloud-based (online) flowmanager only
+* **route:** `/:token/untrusted-settings` where:
+    - `:token` should be the token of the user for which the settings are updated
+* **header:** Authorization: Bearer < access_token >
+    - `access_token` The access token can be first requested via /access_token endpoint. It represents the authorization that grants a GPII app to update settings associated with a token. Refer to [GPII OAuth2 Guide](https://wiki.gpii.net/w/GPII_OAuth_2_Guide#Resource_Owner_GPII_Token_Grant) about the detail steps.
+* **method:** `PUT`
+* **request body:** An object, containing a subset of to-be-updated settings. For example:
```

Comment:
Could we add a little more detail here on the operation of the endpoint: that it merges the incoming preferences with the user's existing preferences?